src/api/public/apidocs-new/paths/source_project_name_cmd_addchannels.yaml

```diff
@@ -0,0 +1,16 @@
+post:
+  summary: Add channels to a source
+  description: Add channels to a source
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  responses:
+    '200':
+      $ref: '../components/responses/succeeded.yaml'
+    '401':
+      $ref: '../components/responses/unauthorized.yaml'
+    '404':
+      $ref: '../components/responses/unknown_project.yaml'
+  tags:
+    - Source
```

Comment:
IIRC we planned not to include all the source endpoints under the same tag, but to have many of them for clarity:

- Sources (for global commands)
- Sources - Projects
- Sources - Packages
- Sources - Files
- **Sources - Project Command (the current one)**
- Sources - Package Command

But @eduardoj could confirm.

Anyway, if we finally keep them all under the same tag, it should be plural: `Sources`.

Reply:
Let's use four groups:
- Sources (for global commands)
- Sources - Projects
- Sources - Packages
- Sources - Files

This will mimic the sections from the current apidocs, and and a new section for global commands. I hope this makes things more clear.

Reply:
```suggestion
    - Sources - Projects
```

Reply:
I'll move the Project Command under Sources - Projects. Is that ok for you @saraycp ?